x-pack/plugins/security_solution/server/lib/telemetry/receiver.ts

```diff
@@ -563,138 +581,188 @@ export class TelemetryReceiver implements ITelemetryReceiver {
+                        bool: {
+                          should: [
+                            {
+                              match_phrase: {
+                                'kibana.alert.rule.name': 'Ransomware Prevention Alert',
                               },
-                            ],
-                          },
+                            },
+                          ],
                         },
                       },
                     },
-                    {
-                      bool: {
-                        must_not: {
-                          bool: {
-                            should: [
-                              {
-                                match_phrase: {
-                                  'kibana.alert.rule.name': 'Ransomware Detection Alert',
-                                },
+                  },
+                  {
+                    bool: {
+                      must_not: {
+                        bool: {
+                          should: [
+                            {
+                              match_phrase: {
+                                'kibana.alert.rule.name': 'Ransomware Detection Alert',
                               },
-                            ],
-                          },
+                            },
+                          ],
                         },
                       },
                     },
-                  ],
-                },
+                  },
+                ],
               },
-              {
-                bool: {
-                  should: [
-                    {
-                      match_phrase: {
-                        'kibana.alert.rule.parameters.immutable': 'true',
-                      },
+            },
+            {
+              bool: {
+                should: [
+                  {
+                    match_phrase: {
+                      'kibana.alert.rule.parameters.immutable': 'true',
                     },
-                  ],
-                },
-              },
-              {
-                range: {
-                  '@timestamp': {
-                    gte: 'now-1h',
-                    lte: 'now',
                   },
+                ],
+              },
+            },
+            {
+              range: {
+                '@timestamp': {
+                  gte: 'now-1h',
+                  lte: 'now',
                 },
               },
-            ],
-          },
-        },
-        aggs: {
-          prebuilt_rule_alert_count: {
-            cardinality: {
-              field: 'event.id',
             },
-          },
+          ],
         },
       },
+      track_total_hits: false,
+      sort: [
+        { '@timestamp': { order: 'asc', format: 'strict_date_optional_time_nanos' } },
+        { _shard_doc: 'desc' },
+      ] as unknown as string[],
+      pit: { id: pitId },
+      search_after: searchAfter,
+      size: 1_000,
```

Comment:
Do we want to keep this a flat constant value or could this benefit from being in the configuration artifact?

Reply:
I'm not sure what value having an external artifact brings to all available\ clusters. If it was targeting individual clusters hardware profiles I would dig it, but not in this case. 1K is small enough for users to not feel an impact.